### DISCUSSION TITLE
If my path is not in usr/local/sbin How to configure

### DISCUSSION THREAD

**Discussion by imaxun** [opencontainers/runc#3695] [Q&A] [answered]
If my path is not in usr/local/sbin How to configure

<img width="1807" alt="48ac9cfabd85a5e9e8caa38aa5e8c29" src="https://user-images.githubusercontent.com/29112991/211503745-a1d5f424-23c8-4a64-b20a-8c7078397f6a.png">

**Comment by imaxun** [comment]
Does anyone know the reason

**Comment by AkihiroSuda** [comment, marked as answer]
Duplicate of https://github.com/containerd/containerd/discussions/7956
